script/configure-web-proxy

```diff
@@ -1,4 +1,26 @@
 #!/bin/sh -e
+
+usage() {
+    cat << EOF
+Usage: configure-web-proxy
+Configure a local web proxy using apache for openQA.
+
+Options:
+ -h, --help         display this help
+EOF
+    exit
+}
+
+opts=$(getopt -o h --long help -n 'parse-options' -- "$@") || usage
+eval set -- "$opts"
+while true; do
+  case "$1" in
+    -h | --help ) usage; shift ;;
+    -- ) shift; break ;;
+    * ) break ;;
+  esac
+done
+
```

Comment:
As long as we only look for a single "-h|--help" I would use the single-line approach as in `script/fetchneedles`, see below, to save some LOC, agreed?

Reply:
Personally i prefer this more verbose style for usage messages.

Reply:
hm, ok. I guess I can keep the PR as is then.

Reply:
This will also make it easier for someone to add a `--nginx` switch ;-)